components/gc-most-requested/_screen-lg-min.scss

```diff
@@ -0,0 +1,11 @@
+/*
+	Most requested component
+*/
+
+// Make sure the component is not inside a container
+main:not(.container) .gc-most-requested,
+div:not(.container) .gc-most-requested {
```

Comment:
This don't seem to work as expected. I moved the "gc-most-requested" inside a component and the style was still applied.

Also, I am unable to observe the effect of that style in FF neither in Chrome. What it does? Should we add an additional working example for that edge case?

Reply:
Alternatively, you can add a style to break bad implementation like undo:
* the h2 float
* col-count set to 1
* No background color.

CSS selector sample: `.container .gc-most-request`